alignak/arbiterlink.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2015-2015: Alignak team, see AUTHORS.txt file for contributors
```

Comment:
nope ;)

the shebang line should only be added to the file aimed to be executed, that is to the daemons only (or any other tool).